app/models/manageiq/providers/openstack/base_metrics_capture.rb

```diff
@@ -26,6 +26,8 @@ def perf_init_openstack
     raise "No EMS defined" if target.ext_management_system.nil?
 
     metering_service, = Benchmark.realtime_block(:connect) do
+      return target.ext_management_system.connect(:service => available_metric_services[metric_service_from_settings]) if \
+        available_metric_services.keys.include? metric_service_from_settings
```

Comment:
If I were looking in the logs trying to figure out why I wasn't seeing metrics, I would want to see that which service was selected, I think.

It's probably worth selecting the service, log (even at debug level) which service was selected, then try to connect to that service.